hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/BlockRecoveryWorker.java

```diff
@@ -628,7 +628,7 @@ public void run() {
                 new RecoveryTaskContiguous(b).recover();
               }
             } catch (IOException e) {
-              LOG.warn("recover Block: {} FAILED: {}", b, e);
+              LOG.warn("recover Block: {} FAILED: ", b, e);
```

Comment:
this modification has nothing to do with this issue, right?

Reply:
yeah, I will submit a new issue to fix this small problem.